mlflow/server/js/src/shared/web-shared/genai-traces-table/GenAITracesTableActions.tsx

```diff
@@ -151,6 +170,13 @@ const TraceActionsDropdown = (props: TraceActionsDropdownProps) => {
           <DropdownMenu.Trigger asChild>{ActionButton}</DropdownMenu.Trigger>
         )}
         <DropdownMenu.Content>
+          <DropdownMenu.Item
+            componentId="mlflow.genai-traces-table.compare-traces"
+            onClick={onCompare}
+            disabled={!canCompare}
+          >
+            {intl.formatMessage({ defaultMessage: 'Compare', description: 'Compare traces button' })}
+          </DropdownMenu.Item>
```

Comment:
let's add a separator here so it's clearer that this is another action
```suggestion
          </DropdownMenu.Item>
          <DropdownMenu.Separator />
```